File: Completo/ws19GUISwing/ws19GUISwing.js/ShoppingCenter/ec.edu.espe.shoppingcenter.model/Customer.js
class Customer {
    constructor(id, firstName, lastName, gender, moneySpent, age, typeOfCustomer, hobbies) {
        this._id = id;
        this._firstName = firstName;
        this._lastName = lastName;
        this._gender = gender;
        this._moneySpent = moneySpent;
        this._age = age;
        this._typeOfCustomer = typeOfCustomer;
        this._hobbies = hobbies;
    }

    getId() { return this._id; }
    setId(value) { this._id = value; }

    getFirstName() { return this._firstName; }
    setFirstName(value) { this._firstName = value; }

    getLastName() { return this._lastName; }
    setLastName(value) { this._lastName = value; }

    getGender() { return this._gender; }
    setGender(value) { this._gender = value; }

    getMoneySpent() { return this._moneySpent; }
    setMoneySpent(value) { this._moneySpent = value; }

    getAge() { return this._age; }
    setAge(value) { this._age = value; }

    getTypeOfCustomer() { return this._typeOfCustomer; }
    setTypeOfCustomer(value) { this._typeOfCustomer = value; }

    getHobbies() { return this._hobbies; }
    setHobbies(value) { this._hobbies = value; }

    toString() {
        return `Customer{id=${this._id}, firstName=${this._firstName}, lastName=${this._lastName}, gender=${this._gender}, moneySpent=${this._moneySpent}, age=${this._age}, typeOfCustomer=${this._typeOfCustomer}, hobbies=[${this._hobbies.join(', ')}]}`;
    }
}

File: Completo/ws20GUI/ws20GUISwing.Java/ws19GUISwing/ShoppingCenter/src/main/java/ec/edu/espe/shoppingcenter/model/Customer.java

package ec.edu.espe.shoppingcenter.model;

import java.util.ArrayList;

/**
 *
 * @author LABS-ESPE
 */
public class Customer {

    public int getId() {
        return id;
    }

    public String getFirstName() {
        return firstName;
    }

    public String getLastName() {
        return lastName;
    }

    public String getGender() {
        return gender;
    }

    public float getMoneySpent() {
        return moneySpent;
    }

    public int getAge() {
        return age;
    }

    public String getTypeOfCustomer() {
        return typeOfCustomer;
    }

    public ArrayList<String> getHobbies() {
        return hobbies;
    }

    public void setId(int id) {
        this.id = id;
    }

    public void setFirstName(String firstName) {
        this.firstName = firstName;
    }

    public void setLastName(String lastName) {
        this.lastName = lastName;
    }

    public void setGender(String gender) {
        this.gender = gender;
    }

    public void setMoneySpent(float moneySpent) {
        this.moneySpent = moneySpent;
    }

    public void setAge(int age) {
        this.age = age;
    }

    public void setTypeOfCustomer(String typeOfCustomer) {
        this.typeOfCustomer = typeOfCustomer;
    }

    public void setHobbies(ArrayList<String> hobbies) {
        this.hobbies = hobbies;
    }

    @Override
    public String toString() {
        return "Customer{" + "id=" + id + ", firstName=" + firstName + ", lastName=" + lastName + ", gender=" + gender + ", moneySpent=" + moneySpent + ", age=" + age + ", typeOfCustomer=" + typeOfCustomer + ", hobbies=" + hobbies + '}';
    }
    int id;
    String firstName;
    String lastName;
    String gender;
    float moneySpent;
    int age;
    String typeOfCustomer;
    ArrayList <String> hobbies;

    public Customer(int id, String firstName, String lastName, String gender, float moneySpent, int age, String typeOfCustomer, ArrayList<String> hobbies) {
        this.id = id;
        this.firstName = firstName;
        this.lastName = lastName;
        this.gender = gender;
        this.moneySpent = moneySpent;
        this.age = age;
        this.typeOfCustomer = typeOfCustomer;
        this.hobbies = hobbies;
    }
    
   
    
    
    
    
    
}
